activestorage/lib/active_storage/analyzer/video_analyzer.rb

```diff
@@ -98,12 +99,22 @@ def streams
         probe["streams"] || []
       end
 
+      def container
+        probe["format"] || {}
+      end
+
       def probe
-        download_blob_to_tempfile { |file| probe_from(file) }
+        @probe_results ||= download_blob_to_tempfile { |file| probe_from(file) }
```

Comment:
Tiny nit: can this be `@probe` to match the method name?

Reply:
[You got it!](https://user-images.githubusercontent.com/771968/91670329-ff4d0400-eae1-11ea-9006-6d2be5ee817a.gif) :+1: